cmd/serverutil/serverutil.go

```diff
@@ -19,19 +19,38 @@ import (
 	"context"
 	"net"
 	"net/http"
+	"strings"
 
 	"github.com/golang/glog"
 	"github.com/grpc-ecosystem/grpc-gateway/runtime"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"google.golang.org/grpc"
 )
 
+// GrpcHandlerFunc returns an http.Handler that delegates to grpcServer on incoming gRPC
+// connections or otherHandler otherwise. Copied from cockroachdb.
+func GrpcHandlerFunc(grpcServer http.Handler, otherHandler http.Handler) http.Handler {
```

Comment:
nit: Go style dictates that "Grpc" should be all uppercase.

```suggestion
func GRPCHandlerFunc(grpcServer http.Handler, otherHandler http.Handler) http.Handler {
```